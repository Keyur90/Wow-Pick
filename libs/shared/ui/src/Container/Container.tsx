import React, { ReactNode } from 'react';
import { Container as MuiContainer } from '@material-ui/core';
import { makeStyles } from '@material-ui/core/styles';

interface ContainerPropTypes {
  children: NonNullable<ReactNode>;
}

const useStyles = makeStyles((theme) => ({
  root: {
    padding: theme.spacing(0, 2),
  },
}));

const Container: React.FC<ContainerPropTypes> = ({ children }) => {
  const classes = useStyles();

  return <MuiContainer classes={classes}>{children}</MuiContainer>;
};

export { Container };
