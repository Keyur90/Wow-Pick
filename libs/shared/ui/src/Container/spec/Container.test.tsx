import React from 'react';
import { render } from '@testing-library/react';
import { Container } from '../Container';

test('renders Container', () => {
  const { container } = render(<Container>Some content</Container>);

  expect(container).toMatchSnapshot();
});
