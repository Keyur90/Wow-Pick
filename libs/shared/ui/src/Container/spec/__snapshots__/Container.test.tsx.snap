// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Container 1`] = `
<div>
  <div
    class="MuiContainer-root makeStyles-root-1 MuiContainer-maxWidthLg"
  >
    Some content
  </div>
</div>
`;
